docs/serverless/08-07-sync-function-with-gitops.md

```diff
@@ -0,0 +1,252 @@
+
+### Install and configure a k3d cluster
+
+1. Install k3d using brew:
+
+  ```bash
+  brew install k3d
+  ```
+
+2. Cluster a default k3d cluster with a single server node:
+
+  ```bash
+  k3d cluster create {CLUSTER-NAME}
+  ```
+
+This command also sets your context to the newly created cluster. Run this command to display the cluster information:
+
+  ```bash
+  kubectl cluster-info
+  ```
+
+3. Apply the Kyma's `functions.serverless.kyma-project.io` and `triggers.eventing.knative.dev` CRDs from the sources in the [`kyma`](https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials/files) repository. You will need them to create the Function and Trigger CRs on the cluster.
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/master/resources/cluster-essentials/files/functions.serverless.crd.yaml && kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/master/resources/cluster-essentials/files/triggers.eventing.knative.dev.crd.yaml
+  ```
+4. Run this command to make sure the CRs were applied:
+
+  ```bash
+  kubectl get customresourcedefinitions
+  ```
+
+### Prepare your local workspace
+
+1. Create a workspace folder in which you will create source files for your Function:
+
+  ```bash
+  mkdir {WORKSPACE_FOLDER}
+  ```
+
+2. Use the `init` Kyma CLI command to create a local workspace with default configuration for a Python Function:
+
+  ```bash
+  kyma init function --runtime python38 --dir $PWD/{WORKSPACE_FOLDER}
+  ```
+
+  > **TIP:** Python 3.8 is only one of the available runtimes. Read about all [supported runtimes and sample Functions to run on them](#details-runtimes).
+
+  This command will download the following files to your workspace folder:
+
+  - `config.yaml`	with the Function's configuration
+  - `handler.py` with the Function's code and the simple "Hello World" logic
+  - `requirements.txt` with an empty file for your Function's custom dependencies
+
+  It will also set **sourcePath** in the `config.yaml` file to the full path of the workspace folder:
+
+  ```yaml
+  name: my-function
+  namespace: default
+  runtime: python38
+  source:
+      sourceType: inline
+      sourcePath: {FULL_PATH_TO_WORKSPACE_FOLDER}
+  ```
+
+### Install and configure Flux
+
+You can now install the Flux operator, connect it with a specific Git repository folder, and authorize Flux to automatically pull changes from this repository folder and apply them on your cluster.
+
+1. Install flux:
+
+  ```bash
+  brew install fluxctl
+  ```
+
+2. Create a `flux` Namespace for the Flux operator's CRDs.
```

Comment:
```suggestion
2. Create a `flux` Namespace for the Flux operator's CRDs:
```

Reply:
You ended previous sentences with a colon so, for consistency sake, I'd use a colon in all the other steps as well.